android/java/org/chromium/chrome/browser/download/settings/BraveDownloadSettings.java

```diff
@@ -35,6 +39,17 @@ public void onCreate(Bundle savedInstanceState) {
                 (ChromeSwitchPreference) findPreference(PREF_AUTOMATICALLY_OPEN_WHEN_POSSIBLE);
         mAutomaticallyOpenWhenPossiblePref.setOnPreferenceChangeListener(this);
 
+        mDownloadProgressNotificationBubblePref =
+                (ChromeSwitchPreference) findPreference(PREF_DOWNLOAD_PROGRESS_NOTIFICATION_BUBBLE);
```

Comment:
should we check for null here ?

Reply:
I don't think it is needed as we are adding this ChromeSwitchPreference from our `brave_download_preferences.xml`